pkg/image/api/helper.go

```diff
@@ -463,6 +463,8 @@ func ImageWithMetadata(image *Image) error {
 			return nil
 		}
 
+		image.DockerImageManifestMediaType = schema1.MediaTypeManifest
```

Comment:
It should be already set for all the schema2 manifests. It's set by the registry on a push and by importer on import. All the code using this field is written with this in mind (empty value is considered as schema v1).